unfetter-discover/unfetter-ui
# manually associate reports

manually associate reports in addition to adding w/ csv

**Test**
1. visit the creation page; https://localhost/#/threat-dashboard/create
2. click circle plus
3. fill out fields in form; add valid links
4. click save
5. optionally import a csv file
6. click continue
7. click save
8. the overview page should show the correct report references on edit and view; https://localhost/#/threat-dashboard
